### DISCUSSION TITLE
[Question] Plans for desktop in 2023?

### DISCUSSION THREAD

**Discussion by mouralabank** [dotnet/maui#13694] [General]
What are the plans for the desktop on .net Maui in 2023? I would like to plan my project.

**Comment by parkgun6** [comment]
- [ ] - - - - [ ] ㅕ76666ㅎㅓ 776ㅊㅅ65ㅠ67ㄱ7ㅠㅅ- ~~.ㅅ~~

**Comment by davidortinau** [comment]
@mouralabank you can find the roadmap here: https://github.com/dotnet/maui/wiki/Roadmap

What might you like us to consider that would be helpful to your project that is desktop specific?

**Reply by mouralabank** [reply]
@davidortinau desktop-focused demands eventually, such as Modal, MDI, toolbar/ribbon, mouse events, keyboard events, options to configure a window (no titlebar, always on top, without minimizing, without maximizing, just maximized...), tabbed pages, control the mouse for accessibility (no clickable areas change the mouse to indicate this, for example), custom dialogs

**Reply by mouralabank** [reply]
https://github.com/dotnet/maui/issues/6210 and https://github.com/dotnet/maui/issues/13777 these two issues are great options when we talk about multi window
